pkgs/development/haskell-modules/configuration-ghc-8.8.x.nix

```diff
@@ -159,14 +146,28 @@ self: super: {
     '';
   });
   system-fileio = doJailbreak super.system-fileio;
-  yaml = self.yaml_0_11_1_2;
+  tasty-hedgehog = doJailbreak super.tasty-hedgehog;
   haskell-src-meta = appendPatch (dontCheck (doJailbreak super.haskell-src-meta)) (pkgs.fetchpatch {
-    url = "https://gitlab.haskell.org/ghc/head.hackage/raw/master/patches/haskell-src-meta-0.8.2.patch";
-    sha256 = "146im1amywyl29kcldvgrxpwj22lrpzxysl7vc8rmn3hrq130dyc";
+    url = "https://gitlab.haskell.org/ghc/head.hackage/raw/master/patches/haskell-src-meta-0.8.3.patch";
+    sha256 = "1asl932mibr5y057xx8v1a7n3qy87lcnclsfh8pbxq1m3iwjkxy8";
   });
   asn1-encoding = appendPatch (dontCheck (doJailbreak super.asn1-encoding)) (pkgs.fetchpatch {
     url = "https://gitlab.haskell.org/ghc/head.hackage/raw/master/patches/asn1-encoding-0.9.5.patch";
     sha256 = "0a3159rnaw6shjzdm46799crd4pxh33s23qy51xa7z6nv5q8wsb5";
   });
   tls = self.tls_1_5_1;
+  vault = dontHaddock super.vault;
+
+  # The test suite does not know how to find the 'alex' binary.
+  alex = overrideCabal (overrideSrc super.alex {
+    src = pkgs.fetchFromGitHub {
```

Comment:
@luc-tielen Could you link to your comment https://github.com/simonmar/alex/issues/140 here so that we know to stop getting `alex` from github when a new release is made?

Reply:
Yes will do, but overrideSrc and overrideCabal are not working well together.. says it can't find alex executable again now. I will fix that tomorrow (and add the comment)